### ISSUE TITLE
Linter says `remark-lint:double-link` and `ToC item not found` when I only have one link.

### ISSUE THREAD

**Issue by Polycarbohydrate** [sindresorhus/awesome-lint#192]
The repository: https://github.com/polycarbohydrate/awesome-tor

I was wondering why these errors are occuring. I don't know why the `applications` link isn't found and why it says double link when I think I have only one link. 

Is this a false positive?

Errors of interest:

```× Linting
README.md:
  ×   5:1  ToC item "Applications" link "https://github.com/Polycarbohydrate/awesome-tor?tab=readme-ov-file#applications" not found  remark-lint:awesome-toc
  ×   5:3  https://github.com/Polycarbohydrate/awesome-tor?tab=readme-ov-file#applications
             remark-lint:double-link
  ×   6:5  https://github.com/Polycarbohydrate/awesome-tor?tab=readme-ov-file#other-applications
             remark-lint:double-link
  ×   7:5  https://github.com/Polycarbohydrate/awesome-tor?tab=readme-ov-file#tails
             remark-lint:double-link
  ×   8:5  https://github.com/Polycarbohydrate/awesome-tor?tab=readme-ov-file#tor--tor-browser
             remark-lint:double-link
  ×   9:3  https://github.com/Polycarbohydrate/awesome-tor?tab=readme-ov-file#Bridges
             remark-lint:double-link
  ×  10:3  https://github.com/Polycarbohydrate/awesome-tor?tab=readme-ov-file#Manual
             remark-lint:double-link```

**Comment by Polycarbohydrate** [comment]
Resolved: https://github.com/sindresorhus/awesome-lint/issues/197
